src/constants.js

```diff
@@ -203,7 +203,7 @@ export const DISEASES = [
 
   {
     name: '2019 novel coronavirus',
```

Comment:
I think this is also technically incorrect, as "2019 novel coronavirus" is the name of the pathogen, not the disease. Let me know if you'd like this fixed as well, @georgiamoon!

Reply:
Yes! The team made a mistake earlier this year when they named it that. Thanks!

Reply:
Okie, just changed it to "Coronavirus disease 2019" for the disease name.